app/models/playlist.rb

```diff
@@ -0,0 +1,11 @@
+class Playlist < ApplicationRecord
+  validates :name, presence: true
+  validate :has_content
+  validates :energy, :danceability, numericality: { greater_than_or_equal_to: 1, less_than_or_equal_to: 100 }
```

Comment:
Plusieurs remarques / itérations possibles ici:

1. Pour que ça soit plus lisible, on peut splitter en 2 lignes, une pour chaque attribut:
```
  validates :energy, numericality: { greater_than_or_equal_to: 1, less_than_or_equal_to: 100 }
  validates :danceability, numericality: { greater_than_or_equal_to: 1, less_than_or_equal_to: 100 }
```

2. Je pense qu'on veut limiter aux entiers:
```
  validates :energy, numericality: { greater_than_or_equal_to: 1, less_than_or_equal_to: 100, only_integer: true }
  validates :danceability, numericality: { greater_than_or_equal_to: 1, less_than_or_equal_to: 100, only_integer: true }
```

3. A la lecture de ton modèle, je comprends qu'on peut avoir le cas de seulement le genre présent. Si c'est le cas, il faut ajouter un `allow_nil: true` en option
```suggestion
  validates :energy, numericality: { greater_than_or_equal_to: 1, less_than_or_equal_to: 100, only_integer: true, allow_nil: true }
  validates :danceability, numericality: { greater_than_or_equal_to: 1, less_than_or_equal_to: 100, only_integer: true, allow_nil: true }
```